R/stan_betareg.fit.R

```diff
@@ -35,7 +35,7 @@ stan_betareg.fit <-
            algorithm = c("sampling", "optimizing", "meanfield", "fullrank"),
            adapt_delta = NULL, 
            QR = FALSE,
-           draws = 1000) {
+           draws = 4000) {
```

Comment:
Is there a reason why this is 4000 for stan_betareg but not stan_glm?

Reply:
Sorry, I was going between more accurate 4000 and faster 1000 (with faster log density computations 4000 would be better for importance sampling, but with slower computations we get some result much faster and then diagnostics recommend to increase draws for more accuracy). I switched this back to 1000. Furthermore, diagnostics and resampling is not yet enabled for betareg.